### PR TITLE
LPS-143305 Poshi Automation for Bug | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-143305

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/stylebookeditor/StyleBookEditor.path [hunk 1]
```diff
@@ -42,6 +42,11 @@
 	<td>xpath=(//button[contains(@class,'style-book-editor__preview-selector')])[${index}]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>PREVIEW_PAGE_UNEDITABLE</td>
+	<td>//div[contains(@style,'cursor: not-allowed')]</td>
+	<td></td>
+</tr>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/stylebookeditor/StyleBookEditor.path:49]: No need

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/stylebooks/StyleBooks.macro [hunk 2]
```diff
@@ -158,6 +158,10 @@ definition {
 			value1 = "${previewType}");
 	}
 
+	macro viewPreviewPageUneditable {
+		AssertElementPresent(locator1 = "StyleBookEditor#PREVIEW_PAGE_UNEDITABLE");
+	}
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/layout/stylebooks/StyleBooks.macro:164]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/stylebooks/StyleBooksWithPreview.testcase [hunk 3]
```diff
@@ -112,6 +112,36 @@ definition {
...
+				themeName = "Minium");
+		}
+
+		task ("Add a style book") {
+			StyleBooksAdmin.openStyleBooksAdmin(siteURLKey = "test-site-name");
+
+			StyleBooksAdmin.addStyleBooks(styleBookName = "Test Style Book Name");
+		}
+
+		task ("View cannot navigate when preview the page") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/stylebooks/StyleBooksWithPreview.testcase:138]: Cannot navigate in preview modal

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/stylebooks/StyleBooksWithPreview.testcase [hunk 4]
```diff
@@ -112,6 +112,36 @@ definition {
...
+			SelectFrame.selectFrameNoLoading(locator1 = "StyleBookEditor#PAGE_PREVIEW_IFRAME");
+
+			StyleBooks.viewPreviewPageUneditable();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/stylebooks/StyleBooksWithPreview.testcase:141]: Click(locator1 = "Home#DEFAULT_LOGO");

AssertElementNotPresent(locator1 = "ProductMenu#TOGGLE");

logoLink

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/stylebooks/StyleBooksWithPreview.testcase [hunk 5]
```diff
@@ -112,6 +112,36 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-143302. The user cannot access the navigation menu when preview a page within a Style Books."
+	@priority = "4"
+	test CannotNavigateWhenPreviewPage {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/stylebooks/StyleBooksWithPreview.testcase:117]: CannotNavigateInPreviewWindow

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/stylebooks/StyleBooksWithPreview.testcase [hunk 6]
```diff
@@ -112,6 +112,36 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-143302. The user cannot access the navigation menu when preview a page within a Style Books."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/stylebooks/StyleBooksWithPreview.testcase:115]: The user cannot navigate in preview window

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, could you update according to comments? Thanks

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/stylebooks/StyleBooksWithPreview.testcase (hunk 4 above)
Comment (ozysouza) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/stylebooks/StyleBooksWithPreview.testcase:141]: Thanks for the heads up!

Comment (ozysouza) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/stylebooks/StyleBooksWithPreview.testcase:141]: @Tim-Cao, after adding these steps you described the bug is reproducible with poshi! After click on the default logo the product menu and other things are shown, it's quite weird.

**Comment by Tim-Cao** [comment]
I see @ozysouza, see changes here
https://github.com/liferay-echo/liferay-portal/pull/7424

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**